aimmo-game-worker/simulation/world_map.py

```diff
@@ -67,7 +72,7 @@ def can_move_to(self, target_location):
             cell = self.get_cell(target_location)
         except KeyError:
             return False
-        return cell.habitable and not cell.avatar
+        return getattr(cell, 'habitable', False) and not getattr(cell, 'avatar', False)
```

Comment:
This method is now a bit non-intuitive to me. "Unknown" (ie: a fogged cell) is returned as False. Proposed alternative behaviour (using None as an unknown flag):
* For unfogged `cell.habitable and not cell.avatar` (previous behaviour).
* For partially fogged: `None if cell.habitable else False`
* For fully fogged: `None`.

Maybe check if it's on the map for fogged cells (if we're giving avatars that info)?